operators/vision/draw_bounding_box.cc

```diff
@@ -0,0 +1,267 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// Note: make sure the input image has the format BGR
+
+#include "draw_bounding_box.hpp"
+#include <algorithm>
+#include <array>
+#include <cmath>
+#include <cassert>
+#include <gsl/narrow>
+#include <gsl/span>
+#include <gsl/span_ext>
+#include <unordered_map>
+#include <vector>
+#include "exceptions.h"
+
+namespace ort_extensions {
+namespace {
+constexpr std::array<std::array<uint8_t, 3>, 10> KBGRColorMap = {{
+    {{0, 0, 255}},      // Red
+    {{0, 255, 0}},      // Green
+    {{255, 0, 0}},      // Blue
+    {{255, 255, 0}},    // Cyan
+    {{255, 0, 255}},    // Magenta
+    {{0, 0, 128}},      // Dark Red/Maroon
+    {{0, 128, 0}},      // Dark Green/Lime
+    {{128, 0, 0}},      // Dark Blue/Navy
+    {{0, 0, 0}},        // Black
+    {{128, 128, 128}},  // Gray
+}};
```

Comment:
'Lime' is 0, 255, 0 according to https://www.htmlcsscolor.com/hex/00ff00 
Yellow and Orange are easily recognized colours that are not used and might be better than black and grey.

Suggested list/names (not ordered):
Red: 0, 0, 255
Yellow: 0, 255, 255
Lime: 0, 255, 0
Cyan: 255, 255, 0
Blue: 255, 0, 0
Magenta: 255, 0, 255
Orange: 0, 128, 255
Maroon: 0, 0, 128
Green: 0, 128, 0
Navy: 128, 0, 0